patch-image.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/bash
+set -ex
+
+# PATCH argument is in the form "project refs" where refs is a list of
+# comma-separated gerrit refs, fo example refs/changes/67/759567/1
```

Comment:
This is a good start, but ideally we need to support more than one project (for depends-on handling).

Reply:
yep, I mentioned that in the commit message